### PR TITLE
Updated article for V9

### PR DESCRIPTION
**Pull request by eshanrnh**
1. Added verified-against: 9.0.0 metadata
2. Added Enable List View step.
3. Fixed some UI menu item names.

### CODE REVIEW

**Comment by sofietoft** [comment]
As far as I remember, Umbraco 9 (the latest beta) ships with several more default media types than Folder, Image and File 🤔 
At least the latest version of Umbraco 8 does, and these changes should have been merged into the latest beta as well.

Just curious - did you test this article against beta 2 or 3?

**Comment by eshanrnh** [comment]
> As far as I remember, Umbraco 9 (the latest beta) ships with several more default media types than Folder, Image and File 🤔
> At least the latest version of Umbraco 8 does, and these changes should have been merged into the latest beta as well.
> 
> Just curious - did you test this article against beta 2 or 3?

Yes, that's right. I'll update the list with the new media types. Also, do I need to include its functionality or will it be taken care of in this [Trello task](https://trello.com/c/rfh5EA0P/66-update-media-types-with-the-new-ones-in-the-814-release)?

I tested the article in both beta 2 and 3.

**Comment by eshanrnh** [comment]
Done 😀

1. Created new article for V9.
2. Added new media types and generalized the functionality of the media types.

File: Fundamentals/Data/Creating-Media/index-v9.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+meta.Description: "Media in Umbraco is handled in much the same way as content. From the backoffice you can upload and create media items, such as images and files."
+versionFrom: 8.14.0
+verified-against: 9.0.0
+---
+
+# Creating media
+
+Media in Umbraco is handled the same way as content. Instead of defining Document Types you define Media Types that acts as a base for media items. The following default Media Types are available:
+
+- Article - TBD
```
Comment (sofietoft) [inline Fundamentals/Data/Creating-Media/index-v9.md:12]: As far I can see, this Media Type allows uploads of `.pfd`, `.docs` and `doc` files.
image

So, perhaps this line could be something like: "Used for uploading and storing documents"? 🤔

**Review by sofietoft — COMMENTED** [review]
Looks great 💪 

I've added a couple of comments.

**Review by sofietoft — APPROVED** [review]

**Pull request merged**